ui/ceval/src/stockfishProtocol.js

```diff
@@ -1,11 +1,17 @@
 var m = require('mithril');
 
+var EVAL_REGEX = new RegExp(''
+  + /^info depth (\d+) seldepth \d+ multipv (\d+) /.source
+  + /score (cp|mate) ([-\d]+) /.source
+  + /(?:(upper|upper)bound )?nodes (\d+) nps (\d+) /.source
+  + /(?:hashfull \d+ )?tbhits \d+ time (\d+) /.source
+  + /pv ?(.*)/.source);
```

Comment:
Fair enough, defensive programming is not bad of course.

Let's assume this can happen (even though I am pretty sure it can not). Would we end up checking `if (!pv) return;` (because below we probably shouldn't set `best` to `''`, for example)? In that case we might be better of not matching in the first place.